bigframes/ml/compose.py

```diff
@@ -234,6 +295,22 @@ def fit(
         self._extract_output_names()
         return self
 
+    # Overwrite the implementation in BaseTransformer, as it only supports the "ML." transformers.
+    # TODO: clarify if this should be changed in base.BaseTransformer?
```

Comment:
Fixing this in the base class would make sense to me. I see no technical reason why BaseTransformer should be restricted to `ML.*` transformers.